azure-spring-boot/src/test/java/com/microsoft/azure/spring/autoconfigure/servicebus/ServiceBusAutoConfigurationTest.java

```diff
@@ -9,158 +9,84 @@
-        System.setProperty(Constants.SUBSCRIPTION_RECEIVE_MODE_PROPERTY, Constants.SUBSCRIPTION_RECEIVE_MODE.name());
-
-        try (AnnotationConfigApplicationContext context = new AnnotationConfigApplicationContext()) {
-            context.register(ServiceBusAutoConfiguration.class);
-            context.refresh();
-        }
-
-        System.clearProperty(Constants.CONNECTION_STRING_PROPERTY);
-        System.clearProperty(Constants.QUEUE_NAME_PROPERTY);
-        System.clearProperty(Constants.QUEUE_RECEIVE_MODE_PROPERTY);
-        System.clearProperty(Constants.TOPIC_NAME_PROPERTY);
-        System.clearProperty(Constants.SUBSCRIPTION_NAME_PROPERTY);
-        System.clearProperty(Constants.SUBSCRIPTION_RECEIVE_MODE_PROPERTY);
+        this.contextRunner.withPropertyValues(Constants.CONNECTION_STRING_PROPERTY, Constants.INVALID_CONNECTION_STRING)
+                .withPropertyValues(Constants.QUEUE_NAME_PROPERTY, Constants.QUEUE_NAME)
+                .withPropertyValues(Constants.QUEUE_RECEIVE_MODE_PROPERTY, Constants.QUEUE_RECEIVE_MODE.name())
+                .withPropertyValues(Constants.TOPIC_NAME_PROPERTY, Constants.TOPIC_NAME)
+                .withPropertyValues(Constants.SUBSCRIPTION_NAME_PROPERTY, Constants.SUBSCRIPTION_NAME)
+                .withPropertyValues(Constants.SUBSCRIPTION_RECEIVE_MODE_PROPERTY,
+                        Constants.SUBSCRIPTION_RECEIVE_MODE.name());
+
+        this.contextRunner.run((context -> assertThat(context).isNotNull()));
     }
 
     @Test
     public void cannotAutowireQueueClientWithInvalidConnectionString() {
-        System.setProperty(Constants.CONNECTION_STRING_PROPERTY, Constants.INVALID_CONNECTION_STRING);
-        System.setProperty(Constants.QUEUE_NAME_PROPERTY, Constants.QUEUE_NAME);
-        System.setProperty(Constants.QUEUE_RECEIVE_MODE_PROPERTY, Constants.QUEUE_RECEIVE_MODE.name());
-
-        verifyBeanCreationException("Failed to instantiate [com.microsoft.azure.servicebus.QueueClient]: " +
-                "Factory method 'queueClient' threw exception; nested exception is " +
-                "com.microsoft.azure.servicebus.primitives.IllegalConnectionStringFormatException: " +
-                "Connection String cannot be parsed.", QueueClient.class);
+        this.contextRunner.withPropertyValues(Constants.CONNECTION_STRING_PROPERTY, Constants.INVALID_CONNECTION_STRING)
+                .withPropertyValues(Constants.QUEUE_NAME_PROPERTY, Constants.QUEUE_NAME)
+                .withPropertyValues(Constants.QUEUE_RECEIVE_MODE_PROPERTY, Constants.QUEUE_RECEIVE_MODE.name());
 
-        System.clearProperty(Constants.CONNECTION_STRING_PROPERTY);
-        System.clearProperty(Constants.QUEUE_NAME_PROPERTY);
-        System.clearProperty(Constants.QUEUE_RECEIVE_MODE_PROPERTY);
+        this.contextRunner.run((context) -> {
+            assertThat(context).doesNotHaveBean(QueueClient.class);
```

Comment:
Can we put the only one statement in one line and remove `{}` of lambda ?

Reply:
Braces removed.